src/tests/unit/tests/reports/components/report-sections/report-body.test.tsx

```diff
@@ -80,43 +122,23 @@ describe('ReportBody', () => {
             ...detailsProps,
         };
 
-        const wrapper = shallow(<ReportBody {...props} />);
-
-        expect(wrapper.getElement()).toMatchSnapshot();
+        const renderResult = render(<ReportBody {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
```

Comment:
After adding below line it generated 60000 lines in the snapshot, can you please suggest me here?
 expectMockedComponentPropsToMatchSnapshots([HeaderSection, DetailsSection, AllOutcomesSummarySection, ResultsContainer, FailedInstancesSection, PassedChecksSection, NotApplicableChecksSection, FooterText]);

Reply:
Without that line we lose *a lot* of information in the snapshot. If there's another way to make sure we aren't losing fidelity I'm open to it but I want to make sure we're still testing everything the old tests did.

Reply:
We could not find any other way to fix the snapshot, so I added your suggested changes.

Reply:
Sorry yes I unresolved and was about to comment and then got pulled away! I figured out a couple things we can do to reduce the size. The snapshots are so much larger because each component is being passed the same exact props so it's repeated information over and over, and because `props.sectionFactory` is getting rendered out fully instead of abbreviated to '[Function]' like it was in the previous snapshots. For this reason, we can check `sectionProps` for a single component and remove `props.sectionFactory` before we snapshot it:

```js
        // Check that sectionProps are as expected for one section since all sections are passed
        // the same props. Remove sectionFactory from the snapshot since it's a large object with
        // redundant information.
        const sectionProps = getMockComponentClassPropsForCall(ReporterHeaderSection);
        expect({ ...sectionProps, sectionFactory: 'Removed for Snapshot' }).toMatchSnapshot(
            'sectionProps',
        );
```